README.md

```diff
@@ -133,6 +133,15 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+
+If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
```

Comment:
One more line break and it's good to go! 😬 

```suggestion
If there is trouble overriding an image or other configurations in ACM, the 
`klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
```

Reply:
Is it correct?